pkg/config/global.go

```diff
@@ -44,6 +44,8 @@ type GlobalOptions struct {
 	AllowNoMatchingRelease bool
 	// logger is the logger to use.
 	logger *zap.SugaredLogger
+	// EnableLiveOutput Show live output from the Helm binary stdout/stderr into Helmfile own stdout/stderr
```

Comment:
Show --> show

Reply:
Thanks! Tweaked in 9e449f6a5aac9cb6d3723be914368cad1cc80c58